src/datasets/arrow_dataset.py

```diff
@@ -2765,17 +2765,28 @@ def to_json(
         batch_size: Optional[int] = None,
         **to_json_kwargs,
     ) -> int:
-        """Exports the dataset to JSON lines.
+        """Export the dataset to JSON or JSON Lines.
```

Comment:
```suggestion
        """Export the dataset to JSON lines or JSON.
```

If the default is jsonl, may be listing it first?